### DISCUSSION TITLE
Whitelisting or prioritizing certain websites

### DISCUSSION THREAD

**Discussion by evva360** [searxng/searxng#3560] [Q&A] [answered]
I am thinking of allowing the users to 'whitelist' their favorite websites, so to the extent possible the answers first are pulled from those websites. If an answer is not available, then it could be opened up to the larger pool. Is it possible to allow something like this within Searxng? I feel rather than blacklists, maintaining whitelists of favorite sites for a particular user could be preferable. I do have my favorite publications or websites as an example.

**Comment by return42** [comment, marked as answer]
> I am thinking of allowing the users to 'whitelist' their favorite websites,
> ..
> Is it possible to allow something like this within SearXNG? 

Not on client side. On server side you can increase priority, see `high_priority:` [Hostnames plugin](https://docs.searxng.org/src/searx.plugins.hostnames.html#hostnames-plugin)
